hoot-core/src/main/cpp/hoot/core/elements/Tags.cpp

```diff
@@ -395,10 +395,10 @@ bool Tags::hasName() const
   return !getName().isEmpty();
 }
 
-bool Tags::haveMatchingName(const Tags& tags1, const Tags& tags2)
+bool Tags::haveMatchingName(const Tags& tags1, const Tags& tags2, const bool strictNameMatch)
 {
-  const QStringList tag1Names = tags1.getNames();
-  const QStringList tag2Names = tags2.getNames();
+  const QStringList tag1Names = tags1.getNames(!strictNameMatch);
+  const QStringList tag2Names = tags2.getNames(!strictNameMatch);
   for (int i = 0; i < tag1Names.size(); i++)
```

Comment:
Found it useful to be able to ignore the alt_name tag in certain situations during joining